btas/generic/coupled_cp_als.h

```diff
@@ -109,14 +111,13 @@ namespace btas{
     /// \param[in] left the reference tensor, $B$ to be decomposed.
     /// \param[in] right the reference tensor, $Z$ to be decomposed.
     /// \param[in] symms the symmetries of the reference tensor.
-    COUPLED_CP_ALS(Tensor& left, Tensor& right, std::vector<int>& symms):
+    COUPLED_CP_ALS(Tensor &left, Tensor &right, std::vector<size_t> &symms) :
```

Comment:
So there aren't const because I reshape them in order to compute a gemm contraction. But I can rewrite the algebra to use the contract function instead allowing these to be const?